bookshelter-app/src/pages/Detail/index.tsx

```diff
@@ -15,22 +14,26 @@ const DetailPage = () => {
 
   useEffect(() => {
     const fetchBookList = async (): Promise<void> => {
```

Comment:
Update naming, this is only fetchBookDetail

Reply:
Fixed!